### DISCUSSION TITLE
Welcome to Openbiblio Discussions!

### DISCUSSION THREAD

**Discussion by Larian77** [Larian77/Openbiblio#17] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by speedy-10** [comment]
Hallo zusammen, 
wir nutzen Openbiblio seit 2014 am Hohenstaufen-Gymnasium und ich freue mich auf die neue Version.
Wäre es möglich, dass ihr mir für eine aktuelle Testinstallation einen Test-Zugang per PN zukommen lässt, damit ich mir das mal anschauen kann?
VG

**Reply by LeAchim59** [reply]
Hallo,
es gibt jetzt eine Beta-Version zum Testen, die man sich bei "Releases" im ZIP-Archiv herunterladen kann.
Viel Spaß beim Testen
Michael

**Comment by speedy-10** [comment]
Hallo, ich würde OpenBiblio gerne in einer VM (LinuxMint) installieren. Nachdem ich die angegebene Anleitung benutzt hatte, wurde ich darauf hingewiesen, dass Xamp unnötig sei. Könnte mir bitte jemand konkret die notwendigen Pakete unter Linux nennen, die ich vorher per `sudo apt get install foo` installieren soll, bevor ich dann mit Openbiblio weitermache.?
VG

**Reply by Larian77** [reply]
Hi

Unter dem Issue den du gepostet hattest hatte ich schon mal einen Link gepostet. Schau mal hier: https://techviewleo.com/how-to-install-lamp-stack-on-linux-mint/?utm_content=cmp-true

VG
Marcus

**Comment by speedy-10** [comment]
Oh, danke, das hat direkt funktioniert.

**Reply by speedy-10** [reply]
Wenn ich jetzt der Anleitung folge, erhalte ich bei Schritt 7.:
```
~$ mysql -uroot mysql
ERROR 1045 (28000): Access denied for user 'root'@'localhost' (using password: NO)

```

**Reply by Larian77** [reply]
Ich gehe mal davon aus dass du bei der Installation von MySQL /MariaDB ein Passwort angelegt hast. Dieses in der Kommandozeile mit angeben also zusätzlich noch -p ‚passwort‘

**Comment by speedy-10** [comment]
~$ mysql -uroot mysql -p "passwort"
mysql  Ver 8.0.36-0ubuntu0.22.04.1 for Linux on x86_64 ((Ubuntu))
Copyright (c) 2000, 2024, Oracle and/or its affiliates.

Oracle is a registered trademark of Oracle Corporation and/or its
affiliates. Other names may be trademarks of their respective
owners.

Usage: mysql [OPTIONS] [database]
  -?, --help          Display this help and exit.
  -I, --help          Synonym for -?
...
load-data-local-dir                       (No default value)
fido-register-factor                      (No default value)
authentication-oci-client-config-profile  (No default value)
oci-config-file                           (No default value)

**Reply by Larian77** [reply]
Ich schrieb doch:
Beim Passwort keine Hochkommas drum, eventuell kein Leerzeichen zwischen -p und dem Passwort. Alternativ wie es in der Ausgabe steht nur ein -p und kein Passwort dahinter, dann wird man nach dem Passwort gefragt

Grundsätzlich bei der Nutzung von Linux:
* vorher ne allgemeine Einführung zu Linux lesen
* Prompts und Ausgaben sorgfältig lesen 
* Linux ist gerade auf der Konsole nicht so einfach zu bedienen wie Windows, ein bisschen Erfahrung sollte man schon haben bzw. jemand dazu holen der dies hat :-)
* Zur Zeit habe ich bissel Zeit hier zu helfen und diese Fragen zu beantworten aber das bisher hat alles nichts mit Openbiblio zu tun, das sind alles noch Grundlagen des Betriebssystems 
* Vielleicht hilft: https://www.computerweekly.com/de/ratgeber/Wie-man-eine-MySQL-Datenbank-unter-Linux-einrichtet

**Reply by speedy-10** [reply]
Hi, vielen Dank für deine Zeit, ich gehe die Anleitung der Reihe nach durch, evtl. könnte man die Anleitung da aktualisieren, wo es dann hakt. Damit können andere wieder leichter einsteigen.

**Comment by speedy-10** [comment]
> 
> 
> ~$ mysql -uroot mysql -p "passwort"
> mysql Ver 8.0.36-0ubuntu0.22.04.1 for Linux on x86_64 ((Ubuntu))
> Copyright (c) 2000, 2024, Oracle and/or its affiliates.
> 
> Oracle is a registered trademark of Oracle Corporation and/or its
> affiliates. Other names may be trademarks of their respective
> owners.
> 
> Usage: mysql [OPTIONS] [database]
> -?, --help Display this help and exit.
> -I, --help Synonym for -?
> ...
> load-data-local-dir (No default value)
> fido-register-factor (No default value)
> authentication-oci-client-config-profile (No default value)
> oci-config-file (No default value)
>

**Reply by Larian77** [reply]
Warum 5x das Gleiche?
Beim Passwort keine Hochkommas drum, eventuell kein Leerzeichen zwischen -p und dem Passwort. Alternativ wie es in der Ausgabe steht nur ein -p, dann wird man nach dem Passwort gefragt

**Comment by speedy-10** [comment]
Hi, leider wurde meine Antwort im Browser nicht als post angezeit, weshalb ich das wiederholte, habe die überflüssigen langen posts gelöscht.
```mysql -uroot mysql -p```funktioniert, danke.
